Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      name: id
+      required: true
+      description: The custom URL category id.
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
+      isArray: false
+      name: folder
+      predefined:
+      - Shared
+      - Mobile Users
+      - Remote Networks
+      - Service Connections
+      - Mobile Users Container
+      - Mobile Users Explicit Proxy
+      required: false
+    - default: false
+      isArray: false
+      name: value
+      required: false
+      description: The custom URL category's value.
+    - auto: PREDEFINED
+      default: false
+      defaultValue: 'false'
+      isArray: false
+      name: overwrite
+      predefined:
+      - 'true'
+      - 'false'
+      description: This argument specifies rather to append or overwrite the....
+    - default: false
+      isArray: false
+      name: description
+      description: The custom URL category's description.
+    - auto: PREDEFINED
+      description: The custom URL category's type.
+      isArray: false
+      name: type
+      predefined:
+      - URL List
+      - Category Match
+      required: false
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Update an existing url category.
+    name: prisma-sase-custom-url-category-update
+    outputs:
+    - contextPath: PrismaSase.CustomURLCategory.id
+      description: The URL category id.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.name
+      description: The URL category name.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.folder
+      description: The URL category folder.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.type
+      description: The URL category type.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.list
+      description: The URL category match list.
+      type: String
+    - contextPath: PrismaSase.CustomURLCategory.description
+      description: The URL category description.
+      type: String
+  - arguments:
+    - default: false
+      isArray: false
+      name: id
+      required: true
+      description: The custom URL category id.
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    execution: false
+    description: Delete a specific url category.
+    name: prisma-sase-custom-url-category-delete
+  - arguments:
+    - default: false
+      description: A specific external dynamic list to return. If not specified, all external dynamic lists will be returned.
+      isArray: false
+      name: id
+      required: false
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASW folder location.
```
Is this a folder location or folder type/name?